mypy.ini

```diff
@@ -90,7 +90,6 @@ exclude = (?x)
    |tests/push/test_push_rule_evaluator.py
    |tests/rest/client/test_transactions.py
    |tests/rest/media/v1/test_media_storage.py
```

Comment:
Can we remove this file too?

Reply:
Unfortunately no. There are many more complex errors.

Reply:
No problem! Just making sure it wasn't an oversight!